secretmanager/regional_samples/create_regional_secret_with_labels.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package regional_secretmanager
+
+// [START secretmanager_create_regional_secret_with_labels]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	secretmanager "cloud.google.com/go/secretmanager/apiv1"
+	"cloud.google.com/go/secretmanager/apiv1/secretmanagerpb"
+	"google.golang.org/api/option"
+)
+
+// createSecretWithLabels creates a new secret with the given name and labels.
+func CreateRegionalSecretWithLabels(w io.Writer, projectId, locationId, id string) error {
```

Comment:
issue: put function in camelCase. We don't export sample functions.

Reply:
I am exporting these so that i can include these in the unit tests